.github/workflows/releases.yml

```diff
@@ -10,13 +10,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
       - name: Set up Go 1.13
         uses: actions/setup-go@v1
         with:
           go-version: 1.13
       - name: Generate changelog
-        run: script/changelog | tee CHANGELOG.md
+        run: |
+          git fetch --unshallow
```

Comment:
This is needed when generating changelog since `checkout@v2` is shallow by default